server/infinispan/src/main/java/org/hawkular/apm/server/infinispan/InfinispanSpanCache.java

```diff
@@ -82,6 +88,13 @@ public Span get(String tenantId, String id) {
 
     @Override
     public void store(String tenantId, List<Span> spans) throws CacheException {
+        store(tenantId, spans, x -> x.getId());
+    }
+
+    @Override
+    public void store(String tenantId, List<Span> spans, Function<Span, String> uniqueIdGenerator)
```

Comment:
Can you rename the parameter `uniqueIdGenerator` to `unused` or `ignored`, to make it clear that this is not used on the overridden method?